src/main/scala/com/cognite/sdk/scala/v1/Client.scala

```diff
@@ -230,7 +230,7 @@ object GenericClient {
       uri"$baseUrl"
     } catch {
       case NonFatal(_) =>
-        throw new IllegalArgumentException(
+        throw new CdfSdkIllegalArgumentException(
```

Comment:
This is not something we should have to catch in Transformations, so I think it's ok to leave it as `IllegalArgumentException`.